.ci/create-release-github.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+OPERATOR_VERSION=$(git describe --tags)
```

Comment:
There is really no need for this script and it can be directly done in the GH action step.